### ISSUE TITLE
Auto save (and copy file) not working and document not trusted

### ISSUE THREAD

**Issue by aliceshack** [mwermelinger/Learn-to-code-for-data-analysis#10]
I've recently been having issues saving a Jupyter notebook after making edits.

I downloaded all these files as a zip and then copied them all into a new folder with file path:
C:\Users\Alice\Documents\Tester

I then open anaconda navigator and went through that to open Jupyter notebooks and opened the file:
Exercises_1
in C:\Users\Alice\Documents\Tester\1_Having_a_go_at_it

When the notebook loads it has a tag at the top saying "Not Trusted" and then after a while it also says "Autosave failed!" and comes up with an orange box which says "Permission Denied: Documents/Tester/1_Having_a_go_at_it/Exercises_1.ipynb". When I try to save the same "Permission Denied..." thing comes up. I've tried clicking on the "Not Trusted" box and said I trust the notebook but it still will not save. When I try to 'save as' it says "Permission denied: Documents/Tester/1_Having_a_go_at_it/Test.ipynb"

I've tried re-downloading the files but this made no difference.

Do you have any idea why this would be and how I could fix it as I can't currently carry on with the course as I am unable to save the notebook.

Many thanks,
Alice

![image](https://user-images.githubusercontent.com/63397108/93817695-3762e500-fc51-11ea-81fb-931b1389823c.png)

**Comment by mwermelinger** [comment]
I'm not a Windows user, but it seems the Tester folder doesn't have the right permissions, or that the files somehow go 'locked' when copied to that folder. Can you open Microsoft Word, write some text and save it to that folder? Is the Tester folder on a different drive than Anaconda? I'll leave this thread open in case any Windows user knows what's the problem, but I'm afraid I can't help further. You seem to be doing everything right.

**Comment by aliceshack** [comment]
I think I have found that the problem was with my antivirus software. It had effectively blacklisted some applications including Jupyter notebooks and R studio, so I wasn't able to make any changes to files. Thankfully I've managed to change this setting. Apparently it's becoming increasingly common in antivirus software to do this if an application can make 'significant' changes so hopefully this comment helps anyone who is having a similar issue!
